Makefile

```diff
@@ -7,7 +7,8 @@ export IMAGE_REPO ?= quay.io/operator-framework/rukpak
 export IMAGE_TAG ?= latest
 IMAGE?=$(IMAGE_REPO):$(IMAGE_TAG)
 KIND_CLUSTER_NAME ?= kind
-KIND := kind
+KIND := kind-load
```

Comment:
See other comment, but I believe you can delete this line.

Reply:
Thanks for the heads up on these two.